itl/plugins-contrib.d/network-components.conf

```diff
@@ -833,3 +833,316 @@ object CheckCommand "printer_health" {
 	vars.printer_health_hostname = "$check_address$"
 	vars.printer_health_mode = "supplies-status"
 }
+
+template CheckCommand "generic-thola-check-command" {
+    import "plugin-check-command"
+
+    command = [ PluginContribDir + "/thola-client" ]
```

Comment:
... here as `command = [ PluginContribDir + "/thola-client", "check" ]`?

Reply:
Is this also possible for the subcommands or is it only possible here for the check command template? Because we are importing this command template in the subcommands and so I don't know how I would need to change them to accomplish that.